packages/compiler/templates/scaffolding.json

```diff
@@ -49,7 +49,7 @@
       "@typespec/http-server-csharp": {
         "description": "CSharp server stubs",
         "options": {
-          "emitter-output-dir": "{output-dir}/server"
+          "emitter-output-dir": "{output-dir}/server/aspnet/generated"
```

Comment:
should also change below the js, the reason I did it this way is because you probably only have one server generation needed

Reply:
aspnet portion is debatable. But the generated part is needed here.